route/settings/SAMPLE-coupled.control

```diff
@@ -76,7 +76,11 @@
 <MCroutedRunoff>        T                          ! kinematic wave routed discharge at reach [L3/T]
```

Comment:
I suggest adding a comment header to this section that says something like

! --------------------
! Route discharge options
! Only one route option can be TRUE, others must be FALSE
! Routing options for types not chosen will be ignored.
!----------------------

Reply:
the note added.